lib/src/main/scala/spinal/lib/bus/localbus/sim/MemBusDriver.scala

```diff
@@ -0,0 +1,43 @@
+package spinal.lib.bus.localbus
+
+import spinal.core._
+import spinal.core.sim._
+
+case class MemBusDriver(bus : MemBus, clockdomain : ClockDomain) {
```

Comment:
All other bus drivers initialize the bus when constructed - this one should do this as well.

Reply:
done